src/components/newsletter.js

```diff
@@ -0,0 +1,14 @@
+import { useRouter } from "next/router";
+import NewsletterSubscriptionButton from "./buttons/newsletter-button";
+
+export default function NewsletterResponses(props) {
+  const router = useRouter();
+  const { user_Id } = router.query;
```

Comment:
I need some advice on this actually. 

Since the link on the email is something like /newsletters/unsubscribe/4564kj5623j4vdsfgsd, access to the last parameter which is the userID is needed. So I figured this was the best way to do it. 

Open to suggestions as to how I can achieve this. @gregv